src/components/WorkOrder/Appointment/AppointmentCalendar.js

```diff
@@ -46,68 +50,93 @@ const AppointmentCalendar = ({ availableAppointments }) => {
     event.preventDefault()
 
     if (isAvailable(date)) {
+      setSlots(getAvailableSlots(date, availableAppointments))
       setSelectedDate(date)
+      setIsdateSelected(true)
     }
   }
 
+  const onCancel = () => {
+    setIsdateSelected(false)
+    setSelectedDate(null)
+  }
+
+  const isOnSummaryPage = () => {
+    setOnSummaryPage(!onSummaryPage)
```

Comment:
Hey this looks like it is setting some sort of state but the function name would indicate this returns a boolean value. Could we update naming perhaps or am I missing something ?